vividus-plugin-web-app/src/test/java/org/vividus/bdd/steps/ui/web/WaitStepsTests.java

```diff
@@ -112,25 +112,24 @@
     @Test
     void testElementByIdDisappears()
     {
+        By locator = By.xpath(".//*[normalize-space(@id)=\"id\"]");
```

Comment:
By.xpath(".//*[normalize-space(@id)=\"id\"]") -> By.xpath(".//*[normalize-space(@id)='id']")

Reply:
The test fails if I change \"id\" to 'id'